airbyte-cdk/java/airbyte-cdk/core/src/main/resources/version.properties

```diff
@@ -1 +1 @@
-version=0.30.11
+version=0.31.0
```

Comment:
Why the minor version bump?

Also please update the CDK changelog.

Reply:
I picked a minor bump because this removed a lot of transitive dependencies.  I'm a little worried that something deep down was relying on on them, but I can't be sure (the tests pass... so yolo?)

Reply:
changelog bumped